pipeline/psth.py

```diff
@@ -0,0 +1,579 @@
+                'trial_condition_func': '_get_trials_exclude_stim',
+                'trial_condition_arg': {
+                    'task': 'audio delay',
+                    'task_protocol': 1,
+                    'outcome': 'hit',
+                    'early_lick': 'no early',
+                    'auto_water': 0,
+                    'free_water': 0,
+                    'trial_instruction': 'right'}
+            },
+            {
+                'trial_condition_name': 'good_noearlylick_left_miss',
+                'trial_condition_func': '_get_trials_exclude_stim',
+                'trial_condition_arg': {
+                    'task': 'audio delay',
+                    'task_protocol': 1,
+                    'outcome': 'miss',
+                    'early_lick': 'no early',
+                    'auto_water': 0,
+                    'free_water': 0,
+                    'trial_instruction': 'left'}
+            },
+            {
+                'trial_condition_name': 'good_noearlylick_right_miss',
+                'trial_condition_func': '_get_trials_exclude_stim',
+                'trial_condition_arg': {
+                    'task': 'audio delay',
+                    'task_protocol': 1,
+                    'outcome': 'miss',
+                    'early_lick': 'no early',
+                    'auto_water': 0,
+                    'free_water': 0,
+                    'trial_instruction': 'right'}
+            },
+            {
+                'trial_condition_name': 'all_noearlylick_nostim',
+                'trial_condition_func': '_get_trials_exclude_stim',
+                'trial_condition_arg': {
+                    '_outcome': 'ignore',
+                    'task': 'audio delay',
+                    'task_protocol': 1,
+                    'early_lick': 'no early',
+                    'auto_water': 0,
+                    'free_water': 0}
+            },
+            {
+                'trial_condition_name': 'all_noearlylick_nostim_left',
+                'trial_condition_func': '_get_trials_exclude_stim',
+                'trial_condition_arg': {
+                    '_outcome': 'ignore',
+                    'task': 'audio delay',
+                    'task_protocol': 1,
+                    'early_lick': 'no early',
+                    'auto_water': 0,
+                    'free_water': 0,
+                    'trial_instruction': 'left'}
+            },
+            {
+                'trial_condition_name': 'all_noearlylick_nostim_right',
+                'trial_condition_func': '_get_trials_exclude_stim',
+                'trial_condition_arg': {
+                    '_outcome': 'ignore',
+                    'task': 'audio delay',
+                    'task_protocol': 1,
+                    'early_lick': 'no early',
+                    'auto_water': 0,
+                    'free_water': 0,
+                    'trial_instruction': 'right'}
+            }
```

Comment:
We do not have an audio delay task
```suggestion
            }
```

Reply:
the content (and perhaps usage) of this table will need to be revisited

Reply:
@xiaow2 could you write down a quick description of PSTHs that you'd like to plot for your task?  Thanks